pyfolio/plotting.py

```diff
@@ -1260,6 +1261,33 @@ def plot_slippage_sensitivity(returns, transactions, positions,
     return ax
 
 
+def plot_capacity_sweep(returns, daily_txn, bt_starting_capital,
+                        min_pv=100000, max_pv=300000000, step_size=1000000,
+                        ax=None):
+
+    captial_base_sweep = pd.Series()
+    for start_pv in xrange(min_pv, max_pv, step_size):
```

Comment:
`xrange` is depracated in python 3. Either use `six.range` or just `range` and eat the memory cost in python 2 which shouldn't be a big deal here.